src/polyfills.js

```diff
@@ -0,0 +1,29 @@
+if (!Array.prototype.find) {
```

Comment:
Acho que nao é papel da lib colocar o Polyfill. 

Acho que so instruir o client de colocar eles mesmo.

No OMS eu coloquei:

```html
<script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=Array.prototype.find|gated"></script>
```

Reply:
Talvez colocar as instruções no README.

Reply:
@klzns Como são muito pequenos, você não acha que vale a pena? Porque só com eles já é garantido IE9+ em todas as funções sem quem tiver que chamar não precisar incluir mais nada (é 427B esse arquivo minificado). To pensando na developer XP que quem usa! hehehe

Reply:
Entendo, mas no render, por exemplo, esses polyfills ja sao inseridos.

Reply:
Eu penso que quando tudo usar o render, esse tipo de detalhe realmente estará resolvido, mas ao meu ver por enquanto vale deixar ali mesmo. Se considerar GZIP estamos falando de 283B e evitar um request a toa

Reply:
Eu concordo que vale a pena incluir nem que precisarmos retirar isso no futuro. É uma adição sem custo alto e que dá um ganho em suporte bem grande

Reply:
concordo que deve ser incluído sim. infelizmente ainda temos poucas coisas no render!